utils/githooks/commit-msg.d/10-watermark

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
-if ! grep 'Required-githooks: true' "$1"; then
+if grep -q '[[:alpha:]]' "$1" && ! grep -q 'Required-githooks: true' "$1"; then
```

Comment:
```suggestion
msg=$(cat $1 | grep -v "^Signed-off-by" | sed -n '/^#.*$/q;p' | grep -v "^$")

if [ -n "$msg" ] && ! grep -q 'Required-githooks: true' "$1"; then
```

Reply:
Hmm, maybe we could even make that sed use Signed-off-by instead since that should be the last thing in the message but regardless this one should be safe....maybe Change-id should be stripped too but I think only Bruno is still using that.